composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_phone/AuthorizationPhoneScreen.kt

```diff
@@ -0,0 +1,251 @@
+
+    LaunchedEffect(component) {
+        component.label.collect { label ->
+            when (label) {
+                AuthorizationPhoneStore.Label.AuthorizationPhoneFailure -> {
+                    state.isError = true
+                    showToast(errorMessage)
+                }
+
+                AuthorizationPhoneStore.Label.AuthorizationPhoneSuccess -> component.onOutput(
+                    AuthorizationPhoneComponent.Output.OpenProfileScreen
+                )
+
+                AuthorizationPhoneStore.Label.OpenProfileAuthorization -> TODO()
+                AuthorizationPhoneStore.Label.ReturnInGoogleAuthorization -> component.onOutput(
+                    AuthorizationPhoneComponent.Output.OpenGoogleScreen
+                )
+            }
+        }
+    }
+
+    AuthorizationPhoneComponent(onEvent = component::onEvent, state)
+}
+
+@Composable
+private fun AuthorizationPhoneComponent(
+    onEvent: (AuthorizationPhoneStore.Intent) -> Unit,
+    state: AuthorizationPhoneStore.State
+) {
+    val elevation = ButtonDefaults.elevation(
+        defaultElevation = 0.dp,
+        pressedElevation = 0.dp,
+        disabledElevation = 0.dp,
+        hoveredElevation = 0.dp,
+        focusedElevation = 0.dp
+    )
+
+    val message = remember { mutableStateOf("") }
+    val closeIcon = remember { mutableStateOf(false) }
+    val focusColor = remember { mutableStateOf(textGrayColor) }
+
+    Column(
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.Top,
+        modifier = Modifier
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .padding(
+                horizontal = 16.dp,
+                vertical = 48.dp
+            ),
+    ) {
+        IconButton(
+            modifier = Modifier.size(size = 48.dp),
+            onClick = {
+                onEvent(AuthorizationPhoneStore.Intent.BackButtonClicked)
+            }) {
+            Icon(
+                imageVector = Icons.Rounded.ArrowBack,
+                tint = Color.Black,
+                contentDescription = "image_back"
+            )
+        }
+
+        Spacer(modifier = Modifier.height(8.dp))
+
+        AuthTabRow(0)
+
+        Column(
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
+        ) {
+            AuthTitle(
+                text = stringResource(MainRes.strings.input_number),
+                modifier = Modifier.padding(bottom = 7.dp),
+                textAlign = TextAlign.Start
+            )
+
+            AuthSubTitle(
+                text = stringResource(MainRes.strings.select_number),
+                modifier = Modifier.padding(bottom = 24.dp),
+                textAlign = TextAlign.Start
+            )
+
+            OutlinedTextField(
+                value = message.value,
+                onValueChange = {
+                    message.value = it
```

Comment:
According to the mvi pattern, you should not change the state here. According to mvikotlin, you have to drop msg into the reducer and change the state there.
Read here: https://arkivanov.github.io/MVIKotlin/store.html